conftest.py

```diff
@@ -23,51 +24,56 @@
 
 
 def pytest_addoption(parser):
-    parser.addoption(
+    frost_parser = parser.getgroup("Frost", "Frost's custom arguments")
```

Comment:
:+1: this will help decoupling things if we want to move more logic to the top level click CLI parser